include/rdma/fi_ext.h

```diff
@@ -129,6 +129,29 @@ struct fid_mem_monitor {
 };
 
 
+/*
+ * Peer provider AV support.
+ */
+struct fid_peer_av;
+
+struct fi_ops_av_owner {
+	size_t	size;
+	int	(*query)(struct fid_peer_av *av, struct fi_av_attr *attr);
+	int	(*lookup)(struct fid_peer_av *av, struct fid_ep *ep,
+			  fi_addr_t *fi_addr);
```

Comment:
Any explanation on how the lookup function works? The signature differs from the regular av lookup op.

Reply:
This takes a local ep as input and returns its address in the AV.  The intent is to get 'my rank'.  We can pick a different name to avoid confusion with the current lookup call.